src/Writers/CSharpWriter/CSharpWriter.Code.cs

```diff
@@ -1037,7 +1037,7 @@ public void Write(IEnumerable<OdcmField> fields)
             }
         }
 
-        public void Write(OdcmField odcmField)
+        public void Write(OdcmProperty odcmField)
```

Comment:
This should just be removed. The CSharp model included Odcm fields for completeness but did not take dependencies on the fields existing, IIRC.